tests/sitemaps/test-class-wpseo-sitemaps-cache.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/**
+ * apply_filters( 'wpseo_enable_xml_sitemap_transient_caching', true ); -- only applicable for retrieval (awesome!)
+ *
+ * Public
+ *
+ * ::register_clear_on_option_update .done
+ * ::clear_transient_cache - can't be done.
+ * ::get_validator_key .done
+ * ::get_storage_key .done
+ * ::clear .done
+ *
+ * Private (determine by coverage)
+ *
+ * ::invalidate_sitemap_cache .implied
+ * ::new_sitemap_cache_validator .implied
+ * ::get_sitemap_cache_validator .implied
+ * ::get_safe_sitemap_cache_type .implied
+ */
+class WPSEO_Sitemaps_Cache_Test extends WPSEO_UnitTestCase {
+
+	/**
+	 * Clean up
+	 */
+	public function tearDown() {
+		parent::tearDown();
+
+		remove_action( 'update_option', array( 'WPSEO_Sitemaps_Cache', 'clear_on_option_update' ) );
+	}
+
+	/**
+	 * Test the building of cache keys
+	 */
+	public function test_get_validator_key_global() {
+		$result = WPSEO_Sitemaps_Cache::get_validator_key();
+
+		$this->assertEquals( 'wpseo_sitemap_cache_validator_global', $result );
```

Comment:
You should use the constant here. If you change the constant you don't want the tests to fail.